0_intro/exercises/fibonacci/src/lib.rs

```diff
@@ -1,5 +1,14 @@
 pub fn fibo(v: u32) -> u32 {
-    todo!()
+    let mut f0: u32 = 0;
+    let mut f1: u32 = 1;
+
+    for i in 1..=v {
```

Comment:
If you don't use the index `i` you can replace it with `_` to make it clear it will not be used.